pyaedt/emit_core/Couplings.py

```diff
@@ -9,12 +9,12 @@ class CouplingsEmit(object):
     """Provides for interaction with the EMIT ```coupling`` folder,
     This class is accessible through the EMIT application results variable
     object (``emit.couplings``).
```

Comment:
```suggestion
    This class is accessible through the results variable
    object (``emit.couplings``) for the EMIT app.
```
Here is a case where it was capitalized, but perhaps this should be rephrased ".... for Emit". We really should standardize on capitalization and whether we use "app" after the product/module name.